### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by EmiLoKiJu**
Project requirements:
Select an API: [https://disease.sh/](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbkxjMHpRbExKRkpWOU1pcVBRcGRtSUNrbTA3QXxBQ3Jtc0tsMWp1SXdSajhRSkF4R3ctdnkwV0xFaG8xdWF3d1NBRE0yYzNGMGVkZjVlaV9YZ09pTkJEcktrSk02VC1CcjZ0OTdPekdmYnRXcTU4cVVFUldKa0dtRUhjZUV3LUdWTXM2eGdVUTBvOVJIZUZqQkI5OA&q=https%3A%2F%2Fdisease.sh%2F&v=eEB3oMOrVAg)
Design: Original design idea by Nelson Sakwa on Behance.
[https://www.behance.net/gallery/31579...](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbmxxWUR6RWVISlZHU3BQWVI1NzFMblJUWjN0UXxBQ3Jtc0ttVXlyZ1BCN25oMFhqcFU2b0I5R1VxcTNxdFFTcUtHQzBuTG00a0hHVkQxTmpxazJEUVMzdHRsSGFJeHpQRm1GcTJJZjg5Rkh6LU9obllZMmY1WjJPak5Mek1fVENHY0p0bHVGRXZteHVOUEQzVUIxUQ&q=https%3A%2F%2Fwww.behance.net%2Fgallery%2F31579789%2FBallhead-App-%28Free-PSDs&v=eEB3oMOrVAg))

Home page
When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
Along with the category name, you will display some numeric values per category that come from the API.
When the user clicks (or taps) on a category item, the application navigates to the details page.

Details page
In the details page, the webapp retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button, the user navigates back

Testing requirements
Only testing the casesSlice functions, mocking store and axios.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,32 +127,64 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+<!-- ACKNOWLEDGEMENTS -->
 
-### Making a Progressive Web App
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Design: [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
+Maps images: [vemaps](https://vemaps.com/)
+Some of the icons: [iconpacks](https://www.iconpacks.net/)
```
Comment (Ibizugbe) [inline README.md]: Nice job on your readme😉. Kindly note that, in fulfillment of the requirements for this milestone, you are meant to acknowledge the author of the design, [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
 Kindly implement that😉.
image

File: src/tests/casesSlice.test.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+          },
+        },
+      ];
+      await store.dispatch(getCases('inputcountry'));
+
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+  });
+
+  describe('getHistoricallData', () => {
```
Comment (Ibizugbe) [inline src/tests/casesSlice.test.js:113]: Nice job writing some test cases for your redux code👍. You are also required to write integration tests for your react components making use of the react-testing-library😉. Kindly do well to implement that👍.
image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @EmiLoKiJu 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] UI looks amazing👍.
- [x] Nice job on the color change feature😉.
- [x] Linters are correctly setup and they are passing👍.
- [x] Project contains some unit tests for some pure functions in the redux👍.
- [x] Test cases are passing👍.
- [x] The project is a single page application and it uses the react-router-dom👍.



## Required Changes ♻️

- [ ] Nice job on your video presentation, It looks great and professional👍. But we are required to ensure the maximum length is 5mins🥲. Your currently exceeds 5mins, Kindly ensure your presentation doesn't exceed 5mins👍.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @EmiLoKiJu , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**